content/configuring/anomaly_scoring.md

```diff
@@ -0,0 +1,151 @@
+Notice that the anomaly score variable name has the suffix `pl1`. Internally, CRS keeps track of anomaly scores on a *per* [*paranoia level*]({{< ref "paranoia_levels" >}} "Page describing paranoia levels.") basis. The individual paranoia level anomaly scores are added together before each round of blocking evaluation takes place, allowing the total combined inbound or outbound score to be compared to the relevant anomaly score threshold.
+
+Tracking the anomaly score per paranoia level allows for clever scoring mechanisms to be employed, such as the [executing paranoia level]({{< ref "paranoia_levels#moving-to-a-higher-paranoia-level" >}} "Section describing the executing paranoia level feature.") feature.
+{{% /notice %}}
+
+The rules files `REQUEST-949-BLOCKING-EVALUATION.conf` and `RESPONSE-959-BLOCKING-EVALUATION.conf` are responsible for executing the inbound (request) and outbound (response) rounds of blocking evaluation, respectively. The rules in these files calculate the total inbound or outbound transactional anomaly score and then make a blocking decision, by comparing the result to the defined threshold and taking blocking action if required.
+
+## Configuring Anomaly Scoring Mode
+
+The following settings can be configured when using anomaly scoring mode:
+
+- Anomaly score thresholds
+- Severity levels
+- Early blocking
+
+If using a native Core Rule Set installation on a web application firewall, these settings are defined in the file `crs-setup.conf`. If running CRS where it has been integrated into a commercial product or CDN then support varies. Some vendors expose these settings in the GUI while other vendors require custom rules to be written which set the necessary variables. Unfortunately, there are also vendors that don't allow these settings to be configured at all.
+
+### Anomaly Score Thresholds
+
+An anomaly score threshold is the cumulative anomaly score at which an inbound request or an outbound response will be blocked.
+
+Most detected inbound threats carry an anomaly score of 5 (by default), while smaller violations, e.g. protocol and standards violations, carry lower scores. An anomaly score threshold of 7, for example, would require multiple rule matches in order to trigger a block (e.g. one "critical" rule scoring 5 plus a lesser-scoring rule, in order to reach the threshold of 7). An anomaly score threshold of 10 would require at least two "critical" rules to match, or a combination of many lesser-scoring rules. **Increasing the anomaly score thresholds makes the CRS less sensitive** and hence less likely to block transactions.
+
+{{% notice warning %}}
+Increasing the anomaly score thresholds may allow some attacks to bypass the CRS rules.
+{{% /notice %}}
+
+{{% notice tip %}}
+A common practice when working with a **new** CRS deployment is to start with very high anomaly score thresholds (even as high as 10000) and gradually lower them over time, through multiple iterations of a tuning process.
```

Comment:
Let me be a bit vain: I was the person who came up with this, and I would not mind being mentioned here. You will probably bring this up in the tuning documentation. In my tutorial 8, I follow a method with 3 rules:

* Always work in blocking mode
* Highest scoring requests go first
* Work in several iterations

People have referred to this as the _Folini Method_.

Reply:
I've changed the wording in that "tip" block. Is it okay?

Reply:
Thank you very much.